src/components/Home.js

```diff
@@ -1,6 +1,49 @@
-import React from 'react';
+import React, { Fragment, Component } from 'react';
+import { Redirect } from 'react-router-dom';
+import Button from '@material-ui/core/Button';
 import '../App.css';
 
-const Home = () => (<h1 className='home'>Home</h1>);
+class Home extends Component {
+
+	constructor(props) {
```

Comment:
You can use state = { redirect: false } instead of constructor. State declaration in constructor needs when you define state value from props.